lib/bh/core_ext/rails/form/check_box_helper.rb

```diff
@@ -19,17 +19,24 @@ def check_box(method, options = {}, checked_value = '1', unchecked_value = '0')
       def check_box_with_block_label(method, options = {}, &block)
         append_class! options, 'form-control' unless inline_form?
         append_class! options, 'checkbox' if horizontal_form?
-        options[:label] ||= method.to_s.humanize
+        options[:label] ||= default_label_for(method)
         base_field method, :checkbox, options, &block
       end
 
       def check_box_with_inline_label(method, options = {}, &block)
         options.merge! offset: true, use_label: false
-        options[:label] ||= method.to_s.humanize
+        options[:label] ||= default_label_for(method)
         base_field method, :checkbox, options do
           label_and_field 'checkbox', method, options, &block
         end
       end
+
+      def default_label_for(method)
+        klass = object.class
+        return method.to_s.humanize unless klass.respond_to?(:human_attribute_name)
+
+        klass.human_attribute_name(method)
```

Comment:
Thanks for your work @atipugin ! I'll look into this `human_attribute_name` method a little (because I don't know it) before accepting.

Meanwhile, do you mind changing the syntax of this method? I don't like `return … if …` statements.
You could just write the following instead:

```ruby
def default_label_for(method)
  if object.class.respond_to? :human_attribute_name
    object.class.human_attribute_name method
  else
    method.to_s.humanize
  end
end
``` 

Apart from that, I want to make check whether:

* "default" is a good name for this method `default_label_for`. What is your rationale behind this name?
* the code will work if `human_attribute_name` is defined, but then no translation exists for the method

Thanks again, everything else looks good!!!